specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json

```diff
@@ -284,6 +284,46 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/privateLinkScopeOperationStatuses/{asyncOperationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get specific operation status": {
+            "$ref": "./examples/privateLinkScopeOperationStatuses.json"
+          }
+        },
+        "tags": [
+          "asyncOperations"
+        ],
+        "operationId": "GetAsyncOperationsStatus",
```

Comment:
Operation ID should be Noun_Verb. Suggest **PrivateLinkScopeOperationStatus_Get**

Reply:
thanks